src/main/java/com/noticemedan/map/model/kdtree/Forest.java

```diff
@@ -20,37 +20,41 @@
 
 	public Forest() {
 		//TODO create different amounts of leafs for zoom levels
-		int[] maxNumberOfElementsAtLeaf = new int[] {100, 100, 100};
+		int[] maxNumberOfElementsAtLeaf = new int[] {100, 100, 100, 100, 100};
 		List<OsmElement> osmElements = this.osmMapData.getOsmElements().toJavaList();
-		OsmElement[][] osmElementArray = new OsmElement[3][];
+		OsmElement[][] osmElementArray = new OsmElement[5][];
 
 		List<OsmElement> zoom0 = new LinkedList<>();
```

Comment:
Giver det ikke mere mening at navngive dem fra 1? Der er vel ikke noget zoom 0 selvom det er ud fra index 0

Reply:
Enig.

Reply:
For mig giver det mere mening med zoom 0 passer til index 0.